examples/example_precompiles_vm/example_precompiles_vm.cpp

```diff
@@ -44,7 +44,7 @@ static evmc_result execute_empty(const evmc_message* msg)
 static evmc_result not_implemented()
 {
     auto result = evmc_result{};
-    result.status_code = EVMC_INTERNAL_ERROR;
+    result.status_code = EVMC_REJECTED;
```

Comment:
We have this special case for non-supported code/message. I think it fits better.

Also we have the specialised `EVMC_PRECOMPILE_FAILURE` which we don't use in this example.